templates/batch-llm/README.md

```diff
@@ -1,27 +1,177 @@
-# LLM offline batch inference with Ray Data and vLLM
```

Comment:
are we gonna keep both README files?

Reply:
This one should be auto-generated from the ipynb using `auto-generate-readme.sh`